### DISCUSSION TITLE
Scanning Achievements & Furniture

### DISCUSSION THREAD

**Discussion by masonlouie** [Andrewthe13th/Inventory_Kamera#444] [Ideas or Feature requests]
Can you please scan Achievements & Furniture?

**Comment by cd36th** [comment]
I would like this furniture feature to be added too, as I am currently making a Google sheet that tracks furniture gift set creation progress, and it would be a lot easier not having to enter furniture quantities manually.

**Comment by manuel-rw** [comment]
Would like to see achievements too!

**Comment by SonGokuSSJ** [comment]
I would also like to have an Achievement Scanning option. Would be nice if that could be exported in a seelie.me json file and paimon.moe json file. There is [AkashaScanner from xenesty](https://github.com/xenesty/AkashaScanner), but he seems to have abandoned the project.
